renderlib/Object3d.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "MathUtil.h"
+
+// an object that can be transformed in 3d space
+class Transform3d
+{
+public:
+  Transform3d()
+  {
+    m_rotation = glm::angleAxis(0.0f, glm::vec3(0.0f, 0.0f, 1.0f));
```

Comment:
Is there a difference between using `angleAxis` and `glm::quat` here?

Reply:
I'll switch it to the simpler version that initializes it with a vec3 of euler angles.